pkg/client/connector/service.go

```diff
@@ -404,11 +403,26 @@ func (s *service) connectWorker(c context.Context, cr *rpc.ConnectRequest, k8sCo
 		Action: "connect",
 	}
 
+	// establish a connection to the daemon gRPC service
+	dlog.Info(c, "Connecting to daemon...")
+	conn, err := client.DialSocket(c, client.DaemonSocketName)
+	if err != nil {
+		dlog.Errorf(c, "unable to connect to daemon: %+v", err)
+		s.cancel()
+		return &rpc.ConnectInfo{
+			Error:     rpc.ConnectInfo_CLUSTER_FAILED,
```

Comment:
Should this be the newly added `ConnectInfo_DAEMON_FAILED`?

Reply:
Oops, yeah it should.